src/CommonITILValidation.php

```diff
@@ -969,201 +969,116 @@ public function showSummary(CommonDBTM $item)
-                    }
-                }
-                echo "<td>" . htmlspecialchars($type_name) . "</td>";
-                echo "<td>" . htmlspecialchars($target_name) . "</td>";
-                $is_answered = $row['status'] !== self::WAITING && $row['users_id_validate'] > 0;
-                echo "<td>" . ($is_answered ? htmlspecialchars(getUserName($row["users_id_validate"])) : '') . "</td>";
-                $comment_validation = RichText::getEnhancedHtml($this->fields['comment_validation'] ?? '', ['images_gallery' => true]);
-                echo "<td><div class='rich_text_container'>" . $comment_validation . "</div></td>";
-
-                $doc_item = new Document_Item();
-                $docs = $doc_item->find(["itemtype"          => $this->getType(),
-                    "items_id"           => $this->getID(),
-                    "timeline_position"  => ['>', CommonITILObject::NO_TIMELINE]
-                ]);
-                 $out = "";
-                foreach ($docs as $docs_values) {
-                     $doc = new Document();
-                     $doc->getFromDB($docs_values['documents_id']);
-                     $out  .= "<a ";
-                     $out .= "href=\"" . Document::getFormURLWithID($docs_values['documents_id']) . "\">";
-                     $out .= htmlspecialchars($doc->getField('name')) . "</a><br>";
-                }
-                 echo "<td>" . $out . "</td>";
+            $doc_item = new Document_Item();
+            $docs = $doc_item->find(["itemtype"          => $this->getType(),
+                "items_id"           => $this->getID(),
+                "timeline_position"  => ['>', CommonITILObject::NO_TIMELINE]
+            ]);
 
-                 echo "</tr>";
+            $document = "";
+            foreach ($docs as $docs_values) {
+                    $doc = new Document();
+                    $doc->getFromDB($docs_values['documents_id']);
+                    $document  .= "<a ";
+                    $document .= "href=\"" . Document::getFormURLWithID($docs_values['documents_id']) . "\">";
+                    $document .= $doc->getField('name') . "</a><br>";
             }
-            echo $header;
-        } else {
-           //echo "<div class='center b'>".__('No item found')."</div>";
-            echo "<tr class='tab_bg_1 noHover'><th colspan='" . $nb_colonnes . "'>";
-            echo __s('No item found') . "</th></tr>\n";
-        }
-        echo "</table>";
 
-        echo "<div id='viewvalidation" . $tID . "$rand'></div>\n";
+            $script = "";
+            if ($canedit) {
+                $params = ['type'             => $this->getType(),
+                    'parenttype'       => static::$itemtype,
+                    static::$items_id  => $this->fields[static::$items_id],
+                    'id'               => $row["id"]
+                ];
+                $script = "<span class='far fa-edit' style='cursor:pointer' title='" . __('Edit') . "' ";
```

Comment:
Cleaner to use heredoc. Would also allow IDE to properly identify these lines as JavaScript.